File: star.cs
using System;
namespace pyramid{
    class star{
        public static void Main(string[] args){
           int m=6,n=6;
            for(int i=0;i<13;i++){
                for(int j=0; j<13;j++){
                    if(j<6 && j>=m || j>=6 && j<=n){               
                    Console.Write(" * ");} else Console.Write("   ");
                }
                Console.WriteLine(" ");
                if(i%2==1){
                    m--;
                    n++;
                }
                
            }
        }
    }
}

File: star.java
public class star {
public static void main(String[] args) {
	int m=6,n=6;
  for(int i=0;i<14;i++) {
	  for(int j=0;j<16;j++) {
		  if(j<=6 &&j>=m || j>=6 && j<=n) {
		  System.out.print(" *");
	  }else System.out.print("  ");
		  }
	  System.out.println();
	  if(i%2==1) {
		  m--;
		  n++;
	  }
  }
}
}
